caches/infinispan/src/main/java/org/commonjava/maven/galley/cache/infinispan/FastLocalCacheProvider.java

```diff
@@ -534,12 +586,13 @@ public boolean delete( ConcreteResource resource )
         final String pathKey = getKeyForPath( nfsFile.getCanonicalPath() );
         synchronized ( getTransfer( resource ) )
         {
+            boolean localDeleted = false;
```

Comment:
Would it make sense to use a while() ... wait() loop here if the file is locked?